dashboards/sonarqube/sonarqube.json

```diff
@@ -294,7 +294,7 @@
 						"nrqlQueries": [
 							{
 								"accountIds": [],
-								"query": "SELECT 100 - (latest(sonarqube_elasticsearch_disk_space_free_bytes) * 100) / latest(sonarqube_elasticsearch_disk_space_total_bytes) AS '(%) used disk space' FROM Metric since 3 months ago"
+								"query": "SELECT 100 - (latest(sonarqube_elasticsearch_disk_space_free_bytes) * 100) / latest(sonarqube_elasticsearch_disk_space_total_bytes) AS '(%) used disk space' FROM Metric"
```

Comment:
```suggestion
								"query": "SELECT (latest(sonarqube_elasticsearch_disk_space_free_bytes) / latest(sonarqube_elasticsearch_disk_space_total_bytes))* 100 AS '(%) used disk space' FROM Metric"
```

Reply:
Hi @mdumpati ,
As per your suggestion, I have made changes. Kindly review it.

Thanks.